### PR TITLE
WIP: Modify docstring in advi().

### PR DESCRIPTION
**Pull request by taku-y**
Modify docstring, although API can be improved as discussed in #1172.

### CODE REVIEW

File: pymc3/variational/advi.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 '''
-Created on Mar 12, 2011
+Created on 2016
```
Comment (twiecki) [inline pymc3/variational/advi.py]: While we're at it: `(c) 2016, John Salvatier & Taku Yoshioka`

File: pymc3/variational/advi.py [hunk 2]
```diff
@@ -32,7 +32,29 @@ def check_discrete_rvs(vars):
 
 def advi(vars=None, start=None, model=None, n=5000, accurate_elbo=False,
     learning_rate=.001, epsilon=.1, random_seed=20090425, verbose=1):
-    """Run ADVI.
+    """Performe automatic differentiation variational inference (ADVI).
```
Comment (twiecki) [inline pymc3/variational/advi.py]: `Perform`

File: pymc3/variational/advi.py [hunk 3]
```diff
@@ -32,7 +32,29 @@ def check_discrete_rvs(vars):
 
 def advi(vars=None, start=None, model=None, n=5000, accurate_elbo=False,
     learning_rate=.001, epsilon=.1, random_seed=20090425, verbose=1):
-    """Run ADVI.
+    """Performe automatic differentiation variational inference (ADVI). 
+
+    This function implements the meanfield ADVI, where the variational 
+    posterior distribution is assumed to spherical Gaussian without
```
Comment (twiecki) [inline pymc3/variational/advi.py]: "to be"

**Comment by twiecki** [comment]
LGTM, still WIP @taku-y?

**Comment by taku-y** [comment]
It's ready to merge, thanks.

**Pull request merged**